spec/requests/registrations_spec.rb

```diff
@@ -441,22 +441,24 @@
     end
 
     context "when signed in as competition manager" do
+      let(:do_add_competition) { FactoryBot.create(:competition, :registration_open, :with_delegate, :visible) }
```

Comment:
I find that name confusing, is it something I'm not getting as a non-native?

Reply:
`do_add` has nothing to do with competitions or OTS in my mind

Reply:
`do_add` is the name of the OTS route in the controller - but I agree it's confusing. Renamed to `ots_competition` - basically just need a variable name that differentiates it from the default competition variable